watertap/tools/parameter_sweep/parameter_sweep.py

```diff
@@ -126,6 +145,18 @@ def __init__(
             h5_parent_group_name=self.config.h5_parent_group_name,
         )
 
+    @staticmethod
+    def _assign_variable_names(model, outputs):
+
+        # Only assign output variable names to unassigned outputs
+        for output_name, _pyo_obj in outputs.items():
+            if not isinstance(_pyo_obj, (_VarData, _ExpressionData, _ParamData)):
+                # Add this object as an expression and assign a name
+                exprs = pyo.Expression(pyo.Any)
+                model.add_component(output_name, exprs)
+                exprs[output_name] = _pyo_obj
+                outputs[output_name] = exprs[output_name]
+
```

Comment:
`from pyomo.common.modeling import unique_component_name`

```suggestion
    @staticmethod
    def _assign_variable_names(model, outputs):

        # Only assign output variable names to unassigned outputs
        exprs = pyo.Expression(pyo.Any)
        model.add_component( unique_component_name(model, "_parameter_sweep_expressions"), exprs )
        for output_name, _pyo_obj in outputs.items():
            if not isinstance(_pyo_obj, (_VarData, _ExpressionData, _ParamData)):
                # Add this object as an expression and assign a name
                exprs[output_name] = _pyo_obj
                outputs[output_name] = exprs[output_name]

```